src/profileData/Matt-code-d.json

```diff
@@ -0,0 +1,12 @@
+{
+    "username": "Matt-code-d",
+    "name": "Matteo Santoro D",
+    "image": "https://i.imgur.com/r0KKbj3.png",
+    "technologiesUsed": "HTML · CSS · JavaScript · WebGL · Construct3",
```

Comment:
This won't work.
Add comma separated values for **technologiesUsed**

Make sure that your portfolio image is **1900x900**.